### PR TITLE
Android dev issue

### PR DESCRIPTION
**Pull request by ertbulbul**
Issue mechanism implemented.

### CODE REVIEW

File: Platon/app/src/main/java/com/cmpe451/platon/page/activity/workspace/fragment/issuedetail/IssueDetailRepository.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                }
+            }
+
+            override fun onFailure(call: Call<Workspace?>, t: Throwable) {
+                call.clone().enqueue(this)
+            }
+
+        })
+    }
+
```
Comment (oykuyilmaz1) [inline Platon/app/src/main/java/com/cmpe451/platon/page/activity/workspace/fragment/issuedetail/IssueDetailRepository.kt:44]: why do we need this? do we use it? maybe workspaceviewmodel can be changed to activityviewmodel and be reused?

File: Platon/app/src/main/res/drawable/ic_add_black_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M19,13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
+      android:fillColor="#000000"/>
```
Comment (darktheorys) [inline Platon/app/src/main/res/drawable/ic_add_black_24dp.xml:8]: our design pattern generally requires "primary_light" for button image. Please double check

File: Platon/app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -29,5 +29,6 @@
     <color name="primary_dark_lighter">#5D6160</color>
     <color name="primary_dark_lighter_transparent">#885D6160</color>
     <color name="secondary_purple2">#B8BED4</color>
+    <color name="very_light_brown">#F8EBD1</color>
```
Comment (darktheorys) [inline Platon/app/src/main/res/values/colors.xml:32]: we use primary light for background colors

**Review by darktheorys — APPROVED** [review]
I have inspected your implementation locally, it is ready to merge with android-dev. It can be imroved via bugfixes. It doesnot crack the current application.

**Pull request merged**